<launch>
    <node pkg="usb_cam" name="usb_cam" type="usb_cam_node">
        <remap from="/usb_cam/image_raw" to="/camera/color/image_raw"/>
    </node>

    <node pkg="motpy_ros" name="face_tracking" type="face_tracking.py" output="screen">
        <param name="weights_url" value="https://github.com/opencv/opencv_3rdparty/raw/dnn_samples_face_detector_20170830/res10_300x300_ssd_iter_140000.caffemodel" />
        <param name="config_url" value="https://raw.githubusercontent.com/opencv/opencv/master/samples/dnn/face_detector/deploy.prototxt" />
        <param name="tracking_size/width" value="640" />
        <param name="tracking_size/height" value="480" />
    </node>

    <node pkg="motpy_ros" name="bbox2arduino" type="bbox2arduino.py">
        <param name="tracking_size/width" value="640" />
        <param name="tracking_size/height" value="480" />
        <param name="imshow_isshow" value="1" />
    </node>

    <node pkg="motpy_ros" name="mosaic" type="mosaic_bbox.py">
        <param name="tracking_size/width" value="640" />
        <param name="tracking_size/height" value="480" />
    </node>

    <node pkg="rosserial_arduino" name="serial_node" type="serial_node.py">
        <param name="port" value="/dev/ttyACM0"/>
    </node>

</launch>